wsmaster/che-core-sql-schema/src/main/resources/che-schema/7.0.0-beta4.0/1__add_devfile.sql

```diff
@@ -0,0 +1,267 @@
+--
+-- SPDX-License-Identifier: EPL-2.0
+--
+-- Contributors:
+--   Red Hat, Inc. - initial API and implementation
+--
+
+-- add devfile table
+CREATE TABLE devfile (
+    id            BIGINT          NOT NULL,
+    spec_version  VARCHAR(255)    NOT NULL,
+    name          VARCHAR(255)    NOT NULL,
+
+    PRIMARY KEY (id)
+);
+
+-- devfile attributes
+CREATE TABLE devfile_attributes (
+    devfile_id     BIGINT,
+    value          TEXT,
+    name           VARCHAR(255)
+);
+-- constraints & indexes
+ALTER TABLE devfile_attributes ADD CONSTRAINT fk_devfile_attributes_devfile_id FOREIGN KEY (devfile_id) REFERENCES devfile (id);
+CREATE UNIQUE INDEX index_devfile_attributes_names ON devfile_attributes (devfile_id, name);
+CREATE INDEX index_devfile_attributes_devfile_id ON devfile_attributes (devfile_id);
+
+-----------------------------------------------------------------------------------------------------------------------------------
+
+-- devfile project
+CREATE TABLE devfile_project (
+    id                   BIGINT       NOT NULL,
+    name                 VARCHAR(255) NOT NULL,
+    clone_path           TEXT,
+    type                 VARCHAR(255) NOT NULL,
+    location             TEXT,
+    branch               VARCHAR(255),
+    start_point          VARCHAR(255),
+    tag                  VARCHAR(255),
+    commit_id            VARCHAR(255),
+    devfile_id           BIGINT,
+
+    PRIMARY KEY (id)
+);
+-- constraints & indexes
+ALTER TABLE devfile_project ADD CONSTRAINT fk_devfile_projects_id FOREIGN KEY (devfile_id) REFERENCES devfile (id);
+CREATE UNIQUE INDEX index_devfile_project_name ON devfile_project (devfile_id, name);
+CREATE INDEX index_projects_devfile_id ON devfile_project (devfile_id);
+
+-----------------------------------------------------------------------------------------------------------------------------------
+
+-- devfile command
+CREATE TABLE devfile_command (
+    id                  BIGINT       NOT NULL,
+    name                VARCHAR(255) NOT NULL,
+    devfile_id          BIGINT,
+
+    PRIMARY KEY (id)
+);
+-- constraints & indexes
+ALTER TABLE devfile_command ADD CONSTRAINT fk_devfile_command_id FOREIGN KEY (devfile_id) REFERENCES devfile (id);
+CREATE UNIQUE INDEX index_devfile_command_name ON devfile_command (devfile_id, name);
+CREATE INDEX index_commands_devfile_id ON devfile_command (devfile_id);
+
+-- devfile command attributes
+CREATE TABLE devfile_command_attributes (
+    devfile_command_id     BIGINT,
+    value                  TEXT,
+    name                   VARCHAR(255)
+);
+-- constraints & indexes
+ALTER TABLE devfile_command_attributes ADD CONSTRAINT fk_devfile_command_attributes_command_id FOREIGN KEY (devfile_command_id) REFERENCES devfile_command (id);
+CREATE UNIQUE INDEX index_devfile_command_attributes_name ON devfile_command_attributes (devfile_command_id, name);
+CREATE INDEX index_command_attributes_command_id ON devfile_command_attributes (devfile_command_id);
```

Comment:
Looks like the comment related to `unique index` and `index` is actual for almost all tables.